pkg/sidecar/util_windows.go

```diff
@@ -0,0 +1,31 @@
+//go:build windows
+// +build windows
+
+package sidecar
+
+import (
+	"context"
+	"errors"
+
+	"github.com/spiffe/go-spiffe/v2/workloadapi"
+)
+
+// RunDaemon starts the main loop
+// Starts the workload API client to listen for new SVID updates
+// When a new SVID is received on the updateChan, the SVID certificates
+// are stored in disk and a restart signal is sent to the proxy's process
+func (s *Sidecar) RunDaemon(ctx context.Context) error {
+	err := workloadapi.WatchX509Context(ctx, &x509Watcher{sidecar: s}, workloadapi.WithNamedPipeName(s.config.AgentAddress))
+	if err != nil && !errors.Is(err, context.Canceled) {
+		return err
+	}
+
+	return nil
+}
+
+func (s *Sidecar) SignalProcess() error {
+	// Signal to reload certs
+	// TODO: it is not possible to get signal by name on windows,
+	// we must provide int here
+	return errors.New("sending signal is not supported on windows")
+}
```

Comment:
So the process can't be reloaded on windows?

Reply:
windows is different to linux, there is no support for `Signal` like on linux,
and only support 
```
	Interrupt Signal = syscall.SIGINT
	Kill      Signal = syscall.SIGKILL
```

So generally what apps are doing on windows is reloading based on another kind of communications, 
like sql scripts or open APIs

Reply:
so, for reloading people can still use run script approach

Reply:
Should we document this?

Reply:
I was thinking about the same. In addition to document that signaling is not supported on Windows, I think that ideally we should check if the configuration has the `renewSignal` setting when running on Windows and fail indicating that is not supported.